schema/url/standalone-map/plugin-collection-markers.raml

```diff
@@ -0,0 +1,81 @@
+#%RAML 1.0 Library
+
+types:
+
+  StandaloneCollectionMapMarkerPostRequest:
+    type: DataPluginRequestBase
+    properties:
+      config: StandaloneCollectionMapMarkerSpec
+
+  StandaloneCollectionMapMarkerSpec:
+    additionalProperties: false
+    description: Specification for a map marker
+    properties:
+      outputEntityId:
+        required: true
+        type: string
+        description: Entity ID associated with collection variable of interest.
+      collectionVariableId:
+        required: true
+        type: VariableSpec
+        description: Collection variable to use for markers.
+      memberVariables:
+        type: MemberVariable[]
+        description: | 
+          Optionally, a set of member variable IDs can be provided. Aggregate output values will be provided per marker
+          for each variable in this set. By default, all variables in collection will be provided. The maximum number of
+          variables we can provide output for per marker is 8.
```

Comment:
i wouldnt mind just requiring they pass a list of 8 or fewer and write some validation, rather than provide values for everything by default.